packages/hotpotqa/src/aviary/envs/hotpotqa/env.py

```diff
@@ -8,9 +8,9 @@
 Agents in the HotPotQA environment can perform searches and lookups to retrieve specific information.
 The environment supports 3 tools:
 
-Search[entity]: Search for a specific entity on Wikipedia and return relevant information.
-Lookup[keyword]: Find and return the next sentence containing the keyword in the current passage.
-SubmitAnswer[answer]: Submit the final answer to the question and conclude the task.
+search[entity]: Search for a specific entity on Wikipedia and return relevant information.
```

Comment:
To share, I think this was like this (proper case names) to match demonstration data